app/src/main/java/com/github/sdpteam15/polyevents/helper/HelperFunctions.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.sdpteam15.polyevents.helper
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import com.github.sdpteam15.polyevents.MainActivity
+import com.github.sdpteam15.polyevents.R
+
+class HelperFunctions private constructor(){
+    companion object{
```

Comment:
Should be an object instead of a class never instanciated containing a companion object